pandas/_libs/tslibs/fields.pyx

```diff
@@ -23,6 +23,26 @@ from np_datetime cimport (pandas_datetimestruct, pandas_timedeltastruct,
 from nattype cimport NPY_NAT
 
 
+def get_time_micros(ndarray[int64_t] dtindex):
+    """
+    Return the number of microseconds in the time component of a
+    nanosecond timestamp.
+
+    Parameters
+    ----------
+    dtindex : ndarray[int64_t]
+
+    Returns
+    -------
+    micros : ndarray[int64_t]
+    """
+    cdef:
+        ndarray[int64_t] micros
+
+    micros = np.mod(dtindex, 86400000000000, dtype=np.int64) // 1000LL
```

Comment:
don't we have this constant defined already somewhere?

Reply:
Yah, it’s defined in conversion. But it’s all the same to the compiler right?  If it was something that was liable to change and needed to be kept in sync (eg nat_strings) it would make sense to cimport it. But for a constant like this it isn’t worth the hassle; it’d be like defining DAYSPERWEEK=7 and importing that all over the place.